chain/indexer/src/lib.rs

```diff
@@ -80,13 +81,18 @@ impl Indexer {
     /// Boots up `near_indexer::streamer`, so it monitors the new blocks with chunks, transactions, receipts, and execution outcomes inside. The returned stream handler should be drained and handled on the user side.
     pub fn streamer(&self) -> mpsc::Receiver<streamer::StreamerMessage> {
         let (sender, receiver) = mpsc::channel(16);
-        actix::spawn(streamer::start(
-            self.view_client.clone(),
-            self.client.clone(),
-            self.near_config.clone(),
-            self.indexer_config.clone(),
-            sender,
-        ));
+        performance_metrics::stats::spawn(
```

Comment:
@frol do you need this for indexer?

Reply:
This is a separate binary, so this is no longer needed.

Reply:
I'll revert this change.